test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/RequestSizeLimitResourceFilterTest.cs

```diff
@@ -0,0 +1,159 @@
+            requestSizeLimitResourceFilterFinal.Bytes = 0;
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { requestSizeLimitResourceFilter, requestSizeLimitResourceFilterFinal });
+
+            var httpMaxRequestBodySize = new TestHttpMaxRequestBodySizeFeature();
+            resourceExecutingContext.HttpContext.Features.Set<IHttpMaxRequestBodySizeFeature>(httpMaxRequestBodySize);
+
+            // Act
+            requestSizeLimitResourceFilter.OnResourceExecuting(resourceExecutingContext);
+            requestSizeLimitResourceFilterFinal.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            Assert.Equal(0, httpMaxRequestBodySize.MaxRequestBodySize);
+            Assert.Equal(1, httpMaxRequestBodySize.Count);
+        }
+
+        [Fact]
+        public void LogsFeatureNotFound()
+        {
+            // Arrange
+            var sink = new TestSink();
+            var loggerFactory = new TestLoggerFactory(sink, enabled: true);
+
+            var requestSizeLimitResourceFilter = new RequestSizeLimitResourceFilter(loggerFactory);
+            requestSizeLimitResourceFilter.Bytes = 12345;
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { requestSizeLimitResourceFilter });
+
+            // Act
+            requestSizeLimitResourceFilter.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            var write = Assert.Single(sink.Writes);
+            Assert.Equal($"A request body size limit could not be applied. This server does not support the IHttpRequestBodySizeFeature.",
+                write.State.ToString());
+        }
+
+        [Fact]
+        public void LogsFeatureIsReadOnly()
+        {
+            // Arrange
+            var sink = new TestSink();
+            var loggerFactory = new TestLoggerFactory(sink, enabled: true);
+
+            var requestSizeLimitResourceFilter = new RequestSizeLimitResourceFilter(loggerFactory);
+            requestSizeLimitResourceFilter.Bytes = 12345;
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { requestSizeLimitResourceFilter });
+
+            var httpMaxRequestBodySize = new TestHttpMaxRequestBodySizeFeature();
+            httpMaxRequestBodySize.IsReadOnly = true;
+            resourceExecutingContext.HttpContext.Features.Set<IHttpMaxRequestBodySizeFeature>(httpMaxRequestBodySize);
+
+            // Act
+            requestSizeLimitResourceFilter.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            var write = Assert.Single(sink.Writes);
+            Assert.Equal($"A request body size limit could not be applied. The IHttpRequestBodySizeFeature for the server is read-only.", write.State.ToString());
+        }
+
+        [Fact]
+        public void RequestSizeLimitAttribute_LogsMaxRequestBodySizeSet()
```

Comment:
Has nothing to do w/ `RequestSizeLimitAttribute`; just call this `LogsMaxRequestBodySizeSet`